xs/src/libslic3r/MedialAxis.cpp

```diff
@@ -1330,6 +1330,33 @@ MedialAxis::grow_to_nozzle_diameter(ThickPolylines& pp, const ExPolygons& anchor
     }
 }
 
+void
+MedialAxis::tapper_ends(ThickPolylines& pp) {
```

Comment:
The word is taper.  Tapper is an old video game :-)

Reply:
erf, my bad.

Reply:
"one of the most addictive games we've seen lately ... not only fun to play, but also immensely challenging, graphically entertaining, and full of action"

it could be worse